src/foam/log/LogLevel.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.ENUM({
+  package: 'foam.log',
+  name: 'LogLevel',
+
+  properties: [
+    {
+      class: 'String',
+      name: 'shortName'
+    },
+    {
+      class: 'String',
+      name: 'consoleMethodName'
+    }
+  ],
+
+  values: [
+    {
+      name: 'DEBUG',
+      shortName: 'DEBG',
+      label: 'Debug',
+      consoleMethodName: 'debug'
+    },
```

Comment:
Wait. I was mistaken. There was a whole conversation about `log` as a log level being a weirdness of Javascript when landing this in the other repo. They `.log()` should be documented as a synonym for `.info()`. That's a documentation error.